perf/ProfileCanvasDiff.jl

```diff
@@ -0,0 +1,390 @@
+# temporarily copied and modified from https://github.com/pfitzseb/ProfileCanvas.jl
```

Comment:
Yeah, totally. We need to decide if we're going to keep this format when we generalize this for the other repos. If so, then we'll definitely condense this like you suggest. I've made a separate issue for this final format #245 , but since this may take a while to figure out, it would be useful to merge this prototype ASAP, so we can start tracking staged runs.